Statements.java

```diff
@@ -1,46 +1,49 @@
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+
 public class Statements {
-    public static void main(String[] args) {
-        byte brain;
-        short steve = 0;
-        int irene = 1;
-        long lex = 4;
-        float fred;
-        double donna;
-        char colin;
-        boolean bob = true;
-        isPrime(irene);
+    private static final Object Prime1000 = getPrime1000;
+
+    public static void main(String[] args)throws Exception
+    {
+        //int irene = 0;
+        //isPrime(irene);
 
         int primeCount = 0;
+        File Prime1000 = new getPrime1000("C:\\Users\\thomas.mcinnes\\Documents\\Prime numbers\\Prime1000.txt");

```

Comment:
I think you might want something along the lines of:

File f = new File("C:\\Users\\thomas.mcinnes\\Documents\\Prime numbers\\Prime1000.txt")